_functions.sh

```diff
@@ -1143,10 +1143,10 @@ do_deploy() {
 
   if [ -z ${DEPLOYMENT_APACHE_VHOST_ALIAS} ]; then
     env_var "DEPLOYMENT_URL" $(do_build_url "http" "${DEPLOYMENT_EXT_HOST}" "${DEPLOYMENT_EXT_PORT}" "")
-    env_var "EXO_HOST_PORT" "${DEPLOYMENT_EXT_HOST}:${DEPLOYMENT_EXT_PORT}"
+    env_var "EXO_HOST_PORT" "${DEPLOYMENT_EXT_HOST}:${DEPLOYMENT_ONLYOFFICE_HTTP_PORT}"
```

Comment:
You should rename the variable to ONLY_OFFICE_HOST_PORT IMO

Reply:
Var renamed